export const getErrorMessage = (error) => error.message || 'There was an error. Please try again later.'
